### DISCUSSION TITLE
Make full "member" data/properties available via Team for the Team's Owners

### DISCUSSION THREAD

**Discussion by bnm91** [cwendt94/espn-api#493] [ESPN API]
With[ this PR](https://github.com/cwendt94/espn-api/pull/487) owner Id were added to Teams.

As a user I'd still like to see the full member data (most importantly **full human names**) on the Team.  I know there's concern about the shape of the member data being unreliable from ESPN but can we try to add it with a lot of safety checks?

**Comment by cwendt94** [comment]
Yeah, I think we can add back the member data and just make sure we handle if its not present. Weird that ESPN removed the data then added it back. I will look at revert this change https://github.com/cwendt94/espn-api/commit/cc82818bada7bda0e12fb10688943fbb46103bd2 and add back the member data
